src/components/ec-table-pagination/ec-table-pagination.vue

```diff
@@ -128,6 +135,14 @@ const props = defineProps({
     type: String,
     default: 'Items per page',
   },
+  isPageSizeHidden: {
```

Comment:
In ec-table.vue we [have](https://github.com/Ebury/chameleon/blob/e5633ff288ce61459b5a075d5cb35711efc32591/src/components/ec-table/ec-table.vue#L84) this default behaviour, I think we should use the same approach here for the compact pagination view.

Reply:
That 768 could be included in the [config](https://github.com/Ebury/chameleon/blob/fc43b627328347d9d66a1988fc9818e303ea7f64/src/config.js#L1-L0) as 
```
screenSizes: {
  md: 768
}
```
@mcibique if you don't mind

Reply:
I thought about doing is as the [isClearFiltersButtonHidden prop](https://github.com/Ebury/chameleon/blob/e5633ff288ce61459b5a075d5cb35711efc32591/src/components/ec-table-filter/ec-table-filter.vue#L66), but it makes more sense to follow that default behavior 

And regarding the config change I think we can do the same for ebo as we have 3 different sizes for `useMediaQuery` there: 1280, 768 and 576 px

Reply:
Yes, I think it makes sense to have all thresholds in screenSizes config property.

Reply:
In this case we won't have flexibility, for example for some cases. We want to show pagination with no hidden parts for md threshold, but want to show customSlot for table rows(some long strings in the table data, so we want to show custom slot earlier than md threshold).

Sticking with tailwind and css we are losing this flexibility.

Reply:
In general, hiding/displaying something based on the media query should be always done via CSS. the page size and total should be always rendered and hidden when reaching a breakpoint.

Reply:
We'll face the case eventually, when we'll have to introduce this props to maintain view from the parent component.

Reply:
You can't configure tailwind from EBO anyway, so passing breakpoints via config will result in some components going after value from TW and some of them using different from a config.

If you want to ignore responsivnes declared in CSS, create one single prop called is-responsive, add a modifier to a component and style the component based on that modifier. See ec-alert using approach like this. ec-alert has also a mixin that allows to redefine the breakpoint if needed.

Reply:
Using a custom class: https://github.com/Ebury/chameleon/blob/master/src/components/ec-alert/ec-alert.story.ts#L66 with https://github.com/Ebury/chameleon/blob/master/src/components/ec-alert/ec-alert.story.css.

Reply:
For smart-table for example this approach could be too restricted or we'll have to define new thresholds for all mixins for subcomponents which leads to know us as users of the smart-table component about it's structure and what it is using under the hood, which what we want to avoid.